Makefile

```diff
@@ -11,9 +11,15 @@ BUILD_FLAGS = -ldflags "-X github.com/tendermint/tendermint/version.GitCommit=`g
 
 all: get_vendor_deps install test
 
-install:
+install: .install_precondition.stamp
```

Comment:
`metalinter` has nothing to do with `install` target. ie. you don't need metalinter in order for install to work

Reply:
it doesn't work without my fix. this hook position works. if you have another place that you think is better please suggest it specifically @melekes   the error without the fix occurs inevitably using the standard build command: ```docker run --rm -v "$PWD":/go/src/github.com/tendermint/tendermint -w /go/src/github.com/tendermint/tendermint -it golang:1.9 make```